src/averageAges.js

```diff
@@ -15,11 +16,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filterCallBack = person =>
+    arguments.length === 2
+      ? Math.ceil(person.died / 100) === century && person.sex === 'm'
+      : person.sex === 'm';
```

Comment:
Lenght of arguments is not the best check here. For example, if function is called like `calculateMenAverageAge(someArray, null)`, arguments lenght will still be two and function will fail

```suggestion
  const filterCallBack = person =>
    century
      ? Math.ceil(person.died / 100) === century && person.sex === 'm'
      : person.sex === 'm';
```